ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,413 @@
+
+```shell
+ansible-avd-examples/     (directory where playbook was run)
+  ├── l2ls-fabric/
+    ├── documentation/
+    ├── group_vars/
+    ├── images/
+    ├── intended/
+    ├── switch-basic-configurations/
+    ├── ansible.cfg
+    ├── inventory.yml
+    ├── playbook.yml
+    └── README.md (this document)
+```
+
+???+ info
+
+    If the content of any file in the example is ***modified*** and the playbook is run again, the file ***will not*** be overwritten. However, if any file in the example is ***deleted*** and the playbook is run again, the file will be re-created.
+
+## Design Overview
+
+### Physical L2LS Topology
+
+The drawing below shows the physical topology used in this example. The interface assignment shown here are referenced across the entire example, so keep that in mind if this example must be adapted to a different topology.
+
+![Figure: 1](images/pure_L2LS_topo.svg)
+
+???+ note
+
+    The FW/L3 Device and individual hosts (A-D) in this example are not managed by AVD, but the switch ports connecting to these devices are.
+
+## Basic EOS Switch Configuration
+
+Basic connectivity between the Ansible controller host and the switches must be established before Ansible can be used to deploy configurations. The following should be configured on all switches:
+
+- Switch Hostname
+- IP enabled interface
+- Username and Password defined
+- Management eAPI Enabled
+
+???+ info
+
+    When using vEOS/cEOS virtual switches, `Management0` is used. When using actual hardware switches, `Management1` is used. The included basic switch configurations may need to be adjusted for your environment.
+
+Below is the basic configuration file for SPINE1:
+
+```shell
+--8<--
+examples/l2ls-fabric/switch-basic-configurations/SPINE1.cfg
+--8<--
+```
+
+## Ansible Inventory
+
+Now that we understand the L2LS topology being used, we need to create the Ansible inventory that respresents this topology. The following is a textual and graphical representation of the Ansible inventory group variables and naming scheme used in this example:
+
+``` text
+- DC1
+  - DC1_FABRIC
+    - DC1_SPINES
+    - DC1_LEAFS
+      - POD1
+      - POD2
+```
+
+DC1 represents the highest level within the hierarchical grouping. Ansible variables defined at this level will be applied to all nodes in the fabric. Ansible groups have parent and child relationships. For example, both DC1_SPINES and DC1_LEAFS are children of DC1_FABRIC. Groups of Groups are possible and allows variables to be shared at any level within the hierarchy. DC1_NETWORK_SERVICES is a group that has 2 other groups defined as children: DC1_SPINES and DC1_LEAFS. The same applies to group named DC1_NETWORK_PORTS. You will see these groups listed at the bottom of the inventory file.
+
+This naming convention makes it possible to easily extend anything, but as always, this can be changed based on your preferences. Just ensure that the names of all groups and hosts are unique.
```

Comment:
```suggestion
This naming convention makes it possible to extend anything quickly, but this can be changed based on your preferences. Just ensure that the names of all groups and hosts are unique.
```